### ISSUE TITLE
Tom Clancy's Ghost Recon® Wildlands (460930)

### ISSUE THREAD

**Issue by NTMan** [ValveSoftware/Proton#1507]
# Compatibility Report
- Name of the game with compatibility issues: Tom Clancy's Ghost Recon® Wildlands
- Steam AppID of the game: 460930

## System Information
- GPU: AMD Vega 56
- Driver/LLVM version: Mesa 18.2.0/7.0.0
- Kernel version:  4.19.0-0.rc3.git2.1.fc30.x86_64 x86_64
- Link to full system information report as [System Info](https://gist.github.com/NTMan/33cca4906328793206218192820779ec)
- Proton version: 3.7-6 Beta

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-460930.log](https://github.com/ValveSoftware/Proton/files/2395305/steam-460930.log)

## Symptoms 
![screenshot from 2018-09-19 08-21-14](https://user-images.githubusercontent.com/200750/45729419-cd3c8300-bbe5-11e8-957f-208ef026a925.png)

[upc.exe_27.0.0.5086_2018-09-19_08-20-56.dmp](https://github.com/ValveSoftware/Proton/files/2395311/upc.exe_27.0.0.5086_2018-09-19_08-20-56.zip)

## Reproduction
Just launch the game.

**Comment by jomarocas** [comment]
same with just dance, i will make a related

**Comment by lucifertdark** [comment]
No games that use Uplay are working in Proton at the moment.

**Comment by Revilum** [comment]
uplay works now with the new proton version  but the game doesn't but it should. It works with the wine tkg 4.4 build https://bugs.winehq.org/show_bug.cgi?id=46880

**Comment by NTMan** [comment]
With Proton 4.2-7 there has been a lot of progress in launching the game.
Yes the game is still crashes, but it happens later after screen of epilepsy warning.

![Screenshot from 2019-06-15 14-40-19](https://user-images.githubusercontent.com/200750/59549795-3a662480-8f7c-11e9-870a-eaefc0ef9719.png)

[steam-460930.log](https://github.com/ValveSoftware/Proton/files/3292979/steam-460930.log)

```
$ inxi -bM
System:    Host: localhost.localdomain Kernel: 5.1.0-1.fc31.x86_64 x86_64 bits: 64 Desktop: Gnome 3.33.2 
           Distro: Fedora release 31 (Rawhide) 
Machine:   Type: Desktop Mobo: ASUSTeK model: ROG STRIX X470-I GAMING v: Rev 1.xx serial: <root required> 
           UEFI: American Megatrends v: 2304 date: 05/03/2019 
CPU:       8-Core: AMD Ryzen 7 2700X type: MT MCP speed: 2128 MHz min/max: 2200/3700 MHz 
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] Vega 20 [Radeon VII] driver: amdgpu v: kernel 
           Display: wayland server: Fedora Project X.org 1.20.5 driver: amdgpu resolution: 3840x2160~60Hz 
           OpenGL: renderer: AMD Radeon VII (VEGA20 DRM 3.30.0 5.1.0-1.fc31.x86_64 LLVM 8.0.0) v: 4.5 Mesa 19.1.0-rc5 
Network:   Device-1: Intel I211 Gigabit Network driver: igb 
           Device-2: Realtek RTL8822BE 802.11a/b/g/n/ac WiFi adapter driver: r8822be 
Drives:    Local Storage: total: 11.57 TiB used: 8.62 TiB (74.5%) 
Info:      Processes: 454 Uptime: 2h 20m Memory: 31.37 GiB used: 11.06 GiB (35.3%) Shell: bash inxi: 3.0.34 
```

**Comment by NTMan** [comment]
Finally, the game start working with Proton 4.11-1
Impressive work, Valve. I appreciate it.

![Screenshot from 2019-07-31 23-13-03](https://user-images.githubusercontent.com/200750/62237630-c3310680-b3ea-11e9-8796-f8ee8bbfd060.jpg)
![Screenshot from 2019-07-31 23-16-05](https://user-images.githubusercontent.com/200750/62237644-c7f5ba80-b3ea-11e9-85f9-1ccdb85c2bd8.jpg)

**Comment by heywoodlh** [comment]
@NTMan, any tweaks you had in order to get it working?

**Comment by NTMan** [comment]
> @NTMan, any tweaks you had in order to get it working?

![Screenshot from 2019-08-03 23-10-29](https://user-images.githubusercontent.com/200750/62415538-b60f5400-b644-11e9-8449-32bad7eab43f.png)

It a weird but without `PROTON_LOG=1` in launch options the game was stuck at the start.

**Comment by macktab** [comment]
Can approve, that game starts only after adding PROTON_LOG=1 (DXVK_HUD is not neccessary).

**Comment by NTMan** [comment]
Let's talk about graphics glitches in Tom Clancy's Ghost Recon Wildlands.
The most notable glitch is when large bushes appear in front of your protagonist then disappear.
![20190805021345_1](https://user-images.githubusercontent.com/200750/62429604-03192600-b72a-11e9-8da6-5c075263dcf9.jpg)
![20190805022631_1](https://user-images.githubusercontent.com/200750/62429605-07454380-b72a-11e9-9605-57787637996f.jpg)
Renderdoc trace is [here](https://mega.nz/#F!9tpnhSTK!btJogpAPWqnNYXXFtquxFg).

**Comment by priand1** [comment]
> Let's talk about graphics glitches in Tom Clancy's Ghost Recon Wildlands.
> The most notable glitch is when large bushes appear in front of your protagonist then disappear.

I am on openSUSE Tumbleweed and had this glitch, too. Installing Mesa-19.2-devel and libllvm9 made it disappear for me.

In my case the game also cannot connect to the Ubisoft-servers. I always get a

ribera-1000b

error. I tried to workaround this with adding 

SSL_CERT_DIR=/etc/ssl/certs

to the launch options but it does not help.
Anyone also experiences this and knows a workaround?

**Comment by priand1** [comment]
Just want to let you know that since proton 4.11.4 wildlands is running perfectly without any workarounds like PROTON_LOG=1 in launch options.

Thanks for all the work to everybody and keep up the good work.

**Comment by NTMan** [comment]
Today I found yet another issue.
When Windows user connect to my game or I connect to Windows user's game then my game stopping with message:
![Screenshot from 2020-12-12 20-54-08](https://user-images.githubusercontent.com/200750/101988824-09c09180-3cbe-11eb-9f22-dfafc55295bd.png)

**Comment by soerengrunewald** [comment]
After the latest uplay update the game crashes at startup.
![Screenshot from 2021-05-01 20-28-52](https://user-images.githubusercontent.com/2650470/116985913-69e7bc80-accd-11eb-9236-d3c482fb51cc.png)
[wildlands_crashdump.txt](https://github.com/ValveSoftware/Proton/files/6420185/wildlands_crashdump.txt)
[steam-460930.log](https://github.com/ValveSoftware/Proton/files/6420211/steam-460930.log)

**Comment by soerengrunewald** [comment]
With [Proton-6.5-GE-2](https://github.com/GloriousEggroll/proton-ge-custom/releases/tag/6.5-GE-2) the game is playable again.

**Comment by NTMan** [comment]
Unfortunately GRW didn't works anymore with Proton-6.5-GE-2 nor 6.10-GE-1.

![Screenshot from 2021-06-11 21-40-42](https://user-images.githubusercontent.com/200750/121726497-12c2dc00-cb04-11eb-8d70-c6a3341b4654.png)

**Comment by spaghetticodez** [comment]
> Unfortunately GRW didn't works anymore with Proton-6.5-GE-2 nor 6.10-GE-1.
> 
> ![Screenshot from 2021-06-11 21-40-42](https://user-images.githubusercontent.com/200750/121726497-12c2dc00-cb04-11eb-8d70-c6a3341b4654.png)

Same issue, not working with any version that i've tested, including GE.

**Comment by kisak-valve** [comment]
Ghost Recon Wildlands

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6020.
@Mr-McBride posted on 2022-07-21T13:15:13:

The game was playable for a day using Proton Experimental, but today when I launch the game it does not recognize any keyboard input.

Is there anything I can do to assist with troubleshooting this?

**Comment by kisak-valve** [comment]
Hello @Mr-McBride, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the issue, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.)

**Comment by alasky17** [comment]
@Mr-McBride also, do you remember approximately what date it was playable with Experimental?

**Comment by Mr-McBride** [comment]
I'm so sorry I missed this. I've changed email providers and completely missed the messages.

To add information. I can completely remove Wildland, then reinstall it and it will work for a bit.

**Comment by Fatmice** [comment]
Does not work.  It will launch to a black screen, then closes with proton experimental
[steam-460930.log](https://github.com/ValveSoftware/Proton/files/9598734/steam-460930.log)

**Comment by agoodside** [comment]
Would like to confirm that the game works on ZorinOS 16.2 pro with Steam installed on Proton: Experimental, 7 and 6. In fact it works great with the exception of multiplayer. Get kicked after about 15 seconds in by EAC (Easy Anti Cheat).

**Comment by kisak-valve** [comment]
Ghost Recon Wildlands (App ID: 460930) crashes on start up

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6634.
@nocnoc94 posted on 2023-03-21T16:58:43:

# Compatibility Report
- Name of the game with compatibility issues: Ghost Recon Wildlands
- Steam AppID of the game: 460930

## System Information
- GPU: RTX 2070 Max Q
- Driver/LLVM version: nvidia 525.89.02
- Kernel version: 6.1.18-200.fc37.x86_64
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/nocnoc94/a9665ff9dcbace9017e10620211f1e21
- Proton version: GE-Proton7-51, 7.0-6

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

### [Proton Logs](https://gist.github.com/nocnoc94/cd79a37dedfc59ca982925da64491345)

## Symptoms
- The game crashes on start up. After the uplay authorization, a small box with the game's logo appears. Then a back window appears and promptly disappears. Steam thinks the game is running but it hangs
- Judging from protonDB, this issue has appeared recently. Prior to this the game was reported to be working 
- [ProtonDB](https://www.protondb.com/app/460930)


## Reproduction
- Open the game and log into uplay with steam

[steam-460930.log](https://github.com/ValveSoftware/Proton/files/11031411/steam-460930.log)

**Comment by nocnoc94** [comment]
The [steam community's fix](https://steamcommunity.com/app/460930/discussions/1/3183487594857196218/) for crashing at launch is to install Visual C++ Redistributable for Visual Studio 2012 which I did using proton tricks but the issue still persists.

I have also cleared uplay's cache which didn't help.

Anyone have any other ideas or knowledge?

**Comment by gabriele2000** [comment]
There's this weird issue where Wildlands will just deadlock when there's a piece of intel that triggers the animation when your character picks up the phone, transfer data from that phone to his phone.

Halfway-through the animation, the game deadlocks, 100% everytime.

**Comment by kisak-valve** [comment]
Hello @gabriele2000, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the deadlock, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by gabriele2000** [comment]
> Hello @gabriele2000, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the deadlock, and attach the generated $HOME/steam-$APPID.log to this issue report as a file. (Proton logs compress well if needed.) Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

Well, I'm using Plain Wine, but I did spot errors coming from "turf effects", although there is no way they're related, like, at all...

[wildlands-log2.zip](https://github.com/ValveSoftware/Proton/files/11594514/wildlands-log2.zip)

https://gist.github.com/gabriele2000/85f19e47d767078d12bc4a7322cc3450

EDIT: Updated log file

**Comment by kisak-valve** [comment]
Second line of the log:

`Please don't report bugs about it on winehq.org and use https://github.com/Frogging-Family/wine-tkg-git/issues instead.`

**Comment by opyate** [comment]
New report.

I bought Wildlands a few years ago, and played it for a couple of hours. I tried to spin it up again now, and it only gets as far as the loading screen. I even left the loading screen running overnight, but the next morning it was still loading.

![ubisoft-ghost-recon-wildlands-loading-screen](https://github.com/ValveSoftware/Proton/assets/96808/6967a72d-93e1-4d8c-b963-fc6b6324bf94)

Here's my Steam -> Help -> System Information: https://gist.github.com/opyate/f11eb78a55b1fd5c10bdc620eaa7954c

Attached is the proton log for Proton Experimental. 

Other things I tried:
- earlier versions of Proton
- Glorious Eggroll versions of Proton
- disabling Steam overlay

Note that this is a new install of Ubuntu 23.10 and I use this one solely for gaming, and all my other games work nicely (Elden Ring, Hogfarts Legacy, Deus Ex: Mankind Divided, Prey 2016, etc etc)

[steam-460930.zip](https://github.com/ValveSoftware/Proton/files/13909098/steam-460930.zip)


Edit: What worked in the end was downgrading the NVidia driver from 535 to 525.
Hopefully this advice helps someone else.

**Comment by jason-eu** [comment]
It's working for me with Proton 9.0-4 on Debain 13.

**Comment by kisak-valve** [comment]
Ghost Recon Wildlands (460930) hangs on splash screen - NVIDIA RTX 4050 / Ubuntu 24.04

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9100.
@MatPhysComp posted on 2025-10-14T01:48:02:

1. Problem Description
The game consistently hangs on its initial splash screen (the title art featuring the soldiers). The Steam client shows the game as "Running", but the application window is frozen and never progresses to the main menu. This occurs after the Ubisoft Connect launcher appears to initialize successfully in the background.

2. System Specifications
OS: Ubuntu 24.04.3 LTS

KERNEL: 6.14.0-33-generic

CPU: Intel Core i7-13620H

GPU: NVIDIA GeForce RTX 4050 Laptop GPU

GPU DRIVER: NVIDIA 580.65.06

RAM: 32 GB

Proton Versions Tested: Proton Experimental (10.0), GE-Proton (10.17, 9.14, 8.32), Proton 8.0-5

3. Troubleshooting Steps Performed
A comprehensive set of troubleshooting steps has been performed without resolving the issue:

Verified integrity of game files via Steam.

Cleared the Steam shader cache.

Completely deleted and regenerated the Proton prefix for the game (compatdata/460930).

Tested with multiple launch options, including PROTON_ENABLE_NVAPI=1 and __NV_PRIME_RENDER_OFFLOAD=1 __GLX_VENDOR_LIBRARY_NAME=nvidia.

Successfully disabled the in-game overlay by launching Ubisoft Connect as a standalone non-Steam game and accessing its internal settings.

Manually repaired the Easy Anti-Cheat service by running EasyAntiCheat_Setup.exe within the game's prefix.

The issue persists regardless of the troubleshooting step performed.

4. Proton Log File
A Proton log was generated using the PROTON_LOG=1 %command% launch option. The end of the log file indicates that the main game process enters an infinite wait/hang state after initialization, without any explicit crash or fatal error messages.

[steam-460930.log](https://github.com/user-attachments/files/22895124/steam-460930.log)

5. Additional Notes
The failure point is consistently after the Ubisoft Connect launcher has finished its tasks and the main game executable, GRW.exe, has taken over. The issue seems to be a deadlock or hang within the game engine itself during its initial loading phase on this specific hardware/software configuration.

**Comment by simifor** [comment]
@MatPhysComp the game is launching fine every time on my amd desktop and nvidia laptop, the nvidia laptop is on a newer driver version with 580.82.9 so that's something worth trying to see if it makes a difference on your end. That aside, it's also worth trying proton experimental bleeding edge, there were changes helping with another ubisoft game that'd also not progress past the initial splash screen in certain configurations https://github.com/ValveSoftware/Proton/wiki/Proton-Versions#proton-bleeding-edge .

**Comment by MatPhysComp** [comment]
Hello @simifor, 

Thanks for the quick response and for confirming the game works on your hardware.

I have followed both suggestions but am still stuck at the initial splash screen:

NVIDIA Driver Update: I successfully updated my driver to 580.95.05 (which is newer than the 580.82.09 version you mentioned).

Proton Version: I also switched to the Proton Experimental bleeding-edge build via the Steam client.

Unfortunately, the game behavior remains the same (stuck/freezing on the initial splash screen).

Are there any known issues regarding the Ubisoft Connect launcher specifically, or any specific launch options (PROTON_USE_WINE=1, etc.) I should try?

Thank you for the help.

**Comment by bastiHST90** [comment]
Hello, im on the latest stable SteamOS on my steam deck OLED.
I bought the game last month, but was only able to try it today.
When I try to start it, the error "Easy Anti-Cheat not found" comes up. When I click OK, the game closes. I can't do anything. (I don't even want to play online). 
I use the default Proton Experimental for this game. I also tried -eac_launcher as launch option. But no dice.

Edit: I think something with Proton Experimental is giving this error. 
I used Proton 10.0-4 now and removed all launch options and it works.

**Comment by kisak-valve** [comment]
Ghost recon Wildlands Freezing

Issue transferred from https://github.com/ValveSoftware/Proton/issues/9479.
@darkhatcher17 posted on 2026-02-09T10:37:53:

# Compatibility Report
- Name of the game with compatibility issues: Ghost Recon Wildlands
- Steam AppID of the game: 460930

## System Information
- GPU: RTX 3050 Mobile
- Video driver version: 590.48.01
- Kernel version: Linux 6.18.8-3-cachyos
- Link to full system information report : https://paste.cachyos.org/p/7a49b5e.log 
- Proton version: Experimental Bleeding Edge-10.0-308760-20260208-p6f42a5-w1d3dc8-d877a15-v3d306e


## Logs: 

[steam-460930.log](https://github.com/user-attachments/files/25180634/steam-460930.log)

## Symptoms

The game runs fine but after around 2 minutes it freezes. The audio is audible but the video is frozen.

## Reproduction

Open the game normally, run it normally and after ~2 minutes the game gets stuck. The issue happens on every proton I have tried so far. Disabled the ubisoft overlay and disabled wayland. I went through the system logs and the NVIDIA driver complains of running out of memory. I have 4 GiB of vram but the game used to run fine like 15 days ago.

[log.log](https://github.com/user-attachments/files/25180677/log.log)

**Comment by darkhatcher17** [comment]
I have been able to run it with Proton Hotfix, haven't saw any crashes/freezes so far.

**Comment by darkhatcher17** [comment]
@kisak-valve After trying too run with the updated proton experimental the issue remains the same. The game still freezes after a while. However the game runs fine with Proton Hotfix for me.

**Comment by alasky17** [comment]
@darkhatcher17 Could you please get a log of the failure on Proton - Experimental as well as a log of the game running successfully on hotfix with the `PROTON_LOG=1 %command%`launch option?  You'll need to rename the log between runs so it doesn't overwrite itself - default logs are stored in ~/steam-[appid].log.

**Comment by darkhatcher17** [comment]
> [@darkhatcher17](https://github.com/darkhatcher17) Could you please get a log of the failure on Proton - Experimental as well as a log of the game running successfully on hotfix with the `PROTON_LOG=1 %command%`launch option? You'll need to rename the log between runs so it doesn't overwrite itself - default logs are stored in ~/steam-[appid].log.

I am extremely sorry for the delay. Got caught up in some work and health issues. 

[steam-460930_experimental.log](https://github.com/user-attachments/files/26001573/steam-460930_experimental.log)

[proton_hotfix.log](https://github.com/user-attachments/files/26001595/proton_hotfix.log)

The hotfix version is old, and it the only one that is working for me. Hope it helps!

**Comment by alasky17** [comment]
@darkhatcher17 Thanks! Unfortunately nothing obvious in the logs, and the game is working here with a similar setup... you could try: 
a) delete the game prefix (by default located in ~/.steam/steam/steamapps/compatdata/460930) after saving a backup in case there are save files or something in there.
b) Deleting your Proton - Experimental install folder (by default located in ~/.steam/steam/steamapps/common/Proton - Experimental) and using "verify integrity of tool file" to redownload - it is possible something is weird there.  
c) Getting rid of any extra config files, launch options, etc.

**Comment by darkhatcher17** [comment]
> [@darkhatcher17](https://github.com/darkhatcher17) Thanks! Unfortunately nothing obvious in the logs, and the game is working here with a similar setup... you could try: a) delete the game prefix (by default located in ~/.steam/steam/steamapps/compatdata/460930) after saving a backup in case there are save files or something in there. b) Deleting your Proton - Experimental install folder (by default located in ~/.steam/steam/steamapps/common/Proton - Experimental) and using "verify integrity of tool file" to redownload - it is possible something is weird there. c) Getting rid of any extra config files, launch options, etc.

Did all that, still happening! the fps drops by a little and then gets stuck. Thanks anyways

**Comment by gordan-ramsey** [comment]
I solved my problem and able to run the game. even with proton 11.
if after using my launcher options you still see a black screen, be patient it takes about 20 seconds on first launch.
The only problem that remains for me is after you quit the game, gamescope stays running with a gray background. you will need to quit it manually (not a big deal to me).

LD_PRELOAD="" gamescope -f -W 3440 -H 1440 -r 60 --force-grab-cursor -- %command% --launcher-skip

<img width="1522" height="769" alt="Image" src="https://github.com/user-attachments/assets/f48d5913-d4ba-4d0c-8d4b-512a61e9c71c" />


-If you don't use --force-grab-cursor then the cursor will escape if your desktop setup has multiple monitors.
-If you don't use --launcher-skip while using --force-grab-cursor the game crashes on startup. when i added this flag "--launcher-skip" it seems that the launcher that comes with the game doesn't like the force grab cursor flag.

Hardware:
3 Monitors (Samung S34E790C, Samsung S32D850T, ASUS VE278) all connected through displayport
CPU: AMD Ryzen 7 3700x
MB: ASUS X570-l
Storage: 2x 2TB M.2 SSDs Sk hynix Platinum P41 SSD PCI-e 4.0.
RAM: 64GB DDR4 @ 3000Mhz
GPU: Nvidia RTX 3060 (Driver Version: 580.173.02 CUDA Version: 13.0)
Mouse: wireless LogiTech M170
OS: Pop! 24.04
Kernel: 7.0.11-76070011-generic

Screenshots from the game:

<img width="3437" height="1440" alt="Image" src="https://github.com/user-attachments/assets/a5b18818-28c1-4a5e-93a7-f51c4be99324" />

<img width="1935" height="793" alt="Image" src="https://github.com/user-attachments/assets/2f7552cb-dce6-4469-8305-5d893b80387f" />
